src/test/java/seedu/address/model/person/GradeListTest.java

```diff
@@ -56,12 +63,12 @@ public void getGrade() {
         gradeList = gradeList.addGrade(thirdGrade);
 
         // same name -> returns true
-        assertEquals(gradeList.getGrade("Midterm"), firstGrade);
-        assertEquals(gradeList.getGrade("Final"), secondGrade);
-        assertEquals(gradeList.getGrade("Assignment"), thirdGrade);
+        assertEquals(gradeList.getGrade("midterm"), firstGrade);
+        assertEquals(gradeList.getGrade("final"), secondGrade);
+        assertEquals(gradeList.getGrade("assignment"), thirdGrade);
 
         // non-existing test name -> returns null
-        assertNull(gradeList.getGrade("Tutorial"));
+        assertNull(gradeList.getGrade("tutorial"));
```

Comment:
After modifying `getGrade` as mentioned, let's update the test cases to include variations of capital cases like "Midterm" and "MIDTERM".